src/static_handler.rs

```diff
@@ -5,6 +5,8 @@ use std::error::Error;
 use std::fmt;
 use time::{self, Timespec};
 
+use std::os::unix::prelude::MetadataExt;
```

Comment:
There are definitely things that look implemented in other platforms but I'm not sure how to have access to them. The deprecation notice at:

https://github.com/rust-lang/rust/blob/1320c293c16dd61093a9b7e8e3c441df6b59ea9e/src/libstd/fs.rs#L1258 did not really help me (cc @alexcrichton). Maybe someone more rusted than I than find the correct `use`.

 is not really helpful (